app/server/lib/Sharing.ts

```diff
@@ -336,7 +337,9 @@ export class Sharing {
       }
       await this._activeDoc.processActionBundle(ownActionBundle);
 
-      const actionSummary = await this._activeDoc.handleTriggers(localActionBundle);
+      const actionSummary = !isCalculate ?
+        await this._activeDoc.handleTriggers(localActionBundle) :
+        createEmptyActionSummary();
```

Comment:
This should use `summarizeAction` like Triggers.ts does, no reason to assume the summary should be empty.